python/chunkText.py

```diff
@@ -23,8 +23,8 @@ def chunkText(scraped_pages: List[Dict[str, str]]) -> List[str]:
     # Returns:
     #   jsons: The list of JSON strings with attributes url and input_text, where input_text is a chunk of the artcile_info_text with length window_size. (Note: return list of JSON strings rather than JSON objects)
     jsons = []
-    window_size = 30
-    window_jump = 10
+    window_size = 100
+    window_jump = 30
```

Comment:
any particular reason?

Reply:
Larger window size means fewer rows are added for testing. But also the size of the chunks didn't seem big enough to contain any meaningful information